transcendental/expokit.f

```diff
@@ -204,8 +218,8 @@ subroutine DMEXPV( n, m, t, v, w, tol, anorm,
 *>>>  break_tol = tol
 *>>>  break_tol = anorm*tol
 
-      call DCOPY( n, v,1, w,1 )
-      beta = DNRM2( n, w,1 )
+      call DCOPYX( n, v,1, w,1 )
```

Comment:
Am I missing a library? I don't see DCOPYX in BLAS, e.g. https://www.google.com/search?q=site%3Anetlib.org+dcopyx

and also

```
ejdavis-m:magicl ejdavis$ objdump -t /usr/local/opt/lapack/lib/libblas.dylib | grep dcopyx
ejdavis-m:magicl ejdavis$ objdump -t /usr/local/opt/openblas/lib/libblas.dylib | grep dcopyx
```